youtube_dl/extractor/vidlii.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 
 import re
+import requests
```

Comment:
Do not use `requests`.

Reply:
Now uses the included urllib instead of requests.